int32 index
uint64 time_stamp
uint8[2097152] data

uint32 MAX_SIZE = 2097152
